pipelines/nuget-package/templates/push.yaml

```diff
@@ -0,0 +1,18 @@
+# about this template: it contains steps needed for pushing the package
+
+# guidance for parameters:
+# - all of parameters should be passed from calling pipeline
+# - all parameters are replaced in the template when the template is loaded = it's a simple replace
+# - therefore all usages needs to be surrounded by ''
+
+parameters:
+  nugetDevOpsFeedToPublish: '! need to be set by pipeline !'
+
+steps:
+- task: DotNetCoreCLI@2
+  displayName: 'NuGet push'
+  inputs:
+    command: push
+    packagesToPush: '$(Build.ArtifactStagingDirectory)/*.symbols.nupkg'
+    publishVstsFeed: '${{ parameters.nugetDevOpsFeedToPublish }}'
+  condition: and(succeeded(), eq(variables['pushpackage'], 'true'))
```

Comment:
Variable 'pushpackage' is not documented. IMHO it can be ommitted since it can be derived from existence of parameter 'nugetDevOpsFeedToPublish'

Reply:
Thanks. I will document the packagepush parameter. 
It can't be ommitted though as it is set via template parameters = exists all the time. Packagepush on the other hand is set during the queue time = can vary from build to build.